# Definition of ROS service LocGetMap for sick localization.
#
# ROS service LocGetMap queries the error status
# by sending cola command ("sMN LocGetMap").
#
# See Telegram-Listing-v1.1.0.241R.pdf for further details about 
# Cola telegrams and this command.

#
# Request (input)
#

---

#
# Response (output)
#

string mappath # The path of the map. This is either relative to maps folder or absolute. Max length = 255
bool success # true: system ready response received from localization controller, false: system not ready or service failed (timeout or error status from controller)

